core/common/src/main/java/alluxio/collections/LockPool.java

```diff
@@ -59,17 +59,7 @@
   private static final String EVICTOR_THREAD_NAME = "LockPool Evictor";
 
   private final Map<K, Resource> mPool;
-  // TODO(baoloongmao): make it configurable before merge this PR.
-  private final ResourcePool<Resource> mResueLockPool = new ResourcePool(1000000) {
```

Comment:
Not sure if this intends to be `reused` or `resource`

Reply:
Sorry, It should be `reuse` as you have guessed what I mean 😄